### DISCUSSION TITLE
Ability to Set Expiration Date for a Discussion

### DISCUSSION THREAD

**Discussion by amanik** [community/community#3867] [Discussions]
As an organization, we want people to respond to a discussion by a certain time. It would be great to specify date and have a discussion expire / or warn people mentioned automatically. I think it'll facilite discussions in a better way. 

As a responder, I can filter by expiring discussions.

**Comment by ghost** [comment]
Hi amanik.

> As an organization, we want people to respond to a discussion by a certain time. It would be great to specify date and have a discussion expire / or warn people mentioned automatically. I think it'll facilite discussions in a better way.
> As a responder, I can filter by expiring discussions.

great idea.
